File: fromJAVAtoPYTHON/04_OOP/03.Instance_Variables_and_Constructors/Book.py
class Book:
    # constructor
    def __init__(self, name) -> None:
        self.name2 = name

    #toString
    def __repr__(self) -> str:
        return repr(self.name2)

#instances
book2 = Book("John")
book3 = Book("Mike")

print(book2)
print(book3)

File: fromJAVAtoPYTHON/04_OOP/03.Instance_Variables_and_Constructors/Book.java
public class Book {
    String name;

    Book(String name) {
        this.name = name;
    }

    public String toString() {
        return name;
    }

    public static void main(String[] args) {
        Book book1 = new Book("John");
        Book book2 = new Book("Mike");

        System.out.println(book1);
        System.out.println(book2);
    }
}